packages/modular-scripts/src/buildPackage.ts

```diff
@@ -693,7 +705,7 @@ export async function buildPackage(
     path.join(modularRoot, outputDirectory, toParamCase(packageName) + '.tgz'),
   );
   /// and... that's it
-  console.log('finished');
+  logger.log('finished');
```

Comment:
Can we add in what the package name is that this is finished for? When building a list of packages/views, it'd be nice to see each finish log tied a package for debugging purposes.

Reply:
Again - `logger` is prefixed so this is already built in.